package.json

```diff
@@ -1,24 +1,57 @@
 {
   "name": "@architect/arc-render-md",
   "version": "0.0.3",
-  "description": "",
-  "main": "renderer.js",
-  "scripts": {
-    "test": "tape 'test/**/*.js' | tap-arc"
-  },
+  "description": "A small stack of Markdown tools configured using the Architect's team preferred conventions.",
   "author": "",
+  "repository": "@architect/arc-render-md",
   "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/architect/arc-render-md/issues"
+  },
+  "type": "module",
+  "scripts": {
+    "dtslint": "dtslint types",
+    "example": "eslint example/index.js --fix && node ./example/index.js",
+    "lint": "eslint src --fix",
+    "tape": "tape test/* | tap-arc",
+    "test": "npm run lint && npm run dtslint && npm run tape"
+  },
+  "main": "./src/index.js",
+  "exports": "./src/index.js",
+  "types": "types",
+  "typings": "types",
+  "files": [
+    "src",
+    "types/index.d.ts"
+  ],
   "dependencies": {
     "@architect/syntaxes": "github:architect/syntaxes#v1.2.1",
     "highlight.js": "^11.5.1",
     "markdown-it": "^13.0.0",
     "markdown-it-external-anchor": "^1.0.0",
-    "markdown-it-toc-and-anchor": "github:macdonst/markdown-it-toc-and-anchor",
-    "querystring": "^0.2.1",
+    "markdown-it-toc-and-anchor": "^4.2.0",
```

Comment:
Oh man, I think we wanna use my fork, `markdown-it-toc-and-anchor` uses `uslug` and it badly impacts coldstart time.

Reply:
Yes I noted above that the fork isn't installable.
Also, does it use uslug even if I pass it the custom slugify function?
I left an item in the Readme's TODO section to possibly use the fork. So I'll merge for now and we can quickly reconsider.

Reply:
That's weird that my fork didn't install for you. Perhaps it needs to be `git+https://github:macdonst/markdown-it-toc-and-anchor.git`?

Yes, even if you pass it a custom slugify function it still does `import uslug from "uslug";` and that import is the problem is `uslug` loads a bunch of arrays of which represent all the unicode characters, IIRC. It's the parsing of the arrays on import during the coldstart which takes forever.

Reply:
It's this command. I believe it needs to be `npm run build`. `yarn` has the build shortcut but npm doesn't
https://github.com/macdonst/markdown-it-toc-and-anchor/blob/master/package.json#L47

That's a bummer about uslug. maybe we can send the original a PR with some benchmarks if we find an equivalent. or with a way to opt out

Reply:
I just committed a change to make it `npm run build`